aws-distro-opentelemetry-node-autoinstrumentation/src/aws-attribute-keys.ts

```diff
@@ -35,4 +36,12 @@ export const AWS_ATTRIBUTE_KEYS: { [key: string]: string } = {
   AWS_BEDROCK_KNOWLEDGE_BASE_ID: 'aws.bedrock.knowledge_base.id',
   AWS_BEDROCK_AGENT_ID: 'aws.bedrock.agent.id',
   AWS_BEDROCK_GUARDRAIL_ID: 'aws.bedrock.guardrail.id',
+  AWS_BEDROCK_GUARDRAIL_ARN: 'aws.bedrock.guardrail.arn',
+  AWS_SNS_TOPIC_ARN: 'aws.sns.topic.arn',
+  AWS_SECRETSMANAGER_SECRET_ARN: 'aws.secretsmanager.secret.arn',
+  AWS_STEPFUNCTIONS_STATEMACHINE_ARN: 'aws.stepfunctions.state_machine.arn',
+  AWS_STEPFUNCTIONS_ACTIVITY_ARN: 'aws.stepfunctions.activity.arn',
+  AWS_LAMBDA_FUNCTION_NAME: 'aws.lambda.function.name',
```

Comment:
Seems this PR is currently missing the feature parity for setting `AWS_LAMBDA_FUNCTION_NAME`